src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/form.html.twig

```diff
@@ -496,12 +500,6 @@
 {% macro render_form_errors(data) %}
     {% if data.errors is defined and data.errors|length > 0 %}
         <div class="errors">
-            <h3>
-                <a class="toggle-button" data-toggle-target-id="{{ data.id }}-errors" href="#">
```

Comment:
why are you removing this toggle button ?

Reply:
This one allowed to toggle the content of the "Errors" tab. 

Tab + toggle seem duplicates to me, and i don't see a case where someone would click on the "errors" tab then would want to hide its content (see capture 1)

But i may have missed something. Did you have something in mind ?